implicit/evaluation.pyx

```diff
@@ -14,7 +14,7 @@ from libc.math cimport fmin
 from libcpp.unordered_set cimport unordered_set
 
 
-def train_test_split(ratings, train_percentage=0.8):
+def train_test_split(ratings, train_percentage=0.8, seed=None):
```

Comment:
Our other API's take a 'random_state' parameter - that is either a np.random.RandomState or an integer (using this function to check/validate https://github.com/benfred/implicit/blob/master/implicit/utils.py#L34 to validate etc).  We should be consistent with other functions here

Reply:
Thanks for that I've missed. I changed the code to use `check_random_state` defined in `implicit/util.py`.